README.md

```diff
@@ -58,7 +58,7 @@ the file.
 If you run `notes something else` on the next day it will still append to the
 same file and continue appending to that file until the next months hits. For
 example, on January 1st 2020 any `notes` commands will append to a
-`2020-01.txt` file.
+`2020-01.txt` file (or whatever filename extension you prefer).
```

Comment:
Can you change the wording of this to: (you can customize the file extension, more on this later).